lascli/parser/users.py

```diff
@@ -18,20 +14,29 @@ def get_user(las_client: Client, user_id):
     return las_client.get_user(user_id)
 
 
-def create_user(las_client: Client, email, app_client_id, **optional_args):
-    avatar = optional_args.get('avatar')
-    if avatar:
-        optional_args['avatar'] = encode_avatar(avatar)
+def create_user(las_client: Client, email, app_client_id, role_ids, **optional_args):
+    args = dict(
+        email=email,
+        app_client_id=app_client_id,
+        **optional_args,
+    )
+    
+    if role_ids:
```

Comment:
Could not `role_ids` be part of `optional_args`?

Reply:
Yes you are right, updated the PR